carbon-design-system/carbon-addons-iot-react
# feat(dashboardeditor): send card select event

Closes #

**Summary**

- Adds a new onCardSelect event to the dashboard editor and fixes a few style issues with the dashboardeditor cards

**Change List (commits, features, bugs, etc)**

- fix(DataSeriesFormContent,TableSeriesFormContent): forced to regen the Combobox if the items change
- feat(DashboardEditor): call the onCardSelect callback when the card is selected (in every case)
- fix(dashboardEditor.scss): fixes the focus outline to not be inside the card
- fix(tableCard): fixes the table title font size and padding to match the other cards

**Acceptance Test (how to verify the PR)**

- Verify that as you select cards in the first dashboard story onCardSelect is triggered
- Verify that the table card focus state works when the Card is in the DashboardEditor, and that the title font-size and padding is correct

**Regression Test (how to make sure this PR doesn't break old functionality)**

- Verify that the normal table title font sizes/paddings don't change



**Things to look for during review**
